SpriteKit2/Final/RickAndMorty/Scenes/Characters/EmojiLoadingScene.swift

```diff
@@ -0,0 +1,58 @@
+//
+//  EmojiLoadingScene.swift
+//  RickAndMorty
+//
+//  Created by Matej Molnár on 25.11.2022.
+//
+
+import SpriteKit
+
+class EmojiLoadingScene: SKScene {
+    override func didMove(to view: SKView) {
+        backgroundColor = .clear
+        
+        setupEmojis([
+            Assets.Image.Emojis.rick,
+            Assets.Image.Emojis.morty,
+            Assets.Image.Emojis.meeseeks,
+            Assets.Image.Emojis.pickle
+        ])
+        startAnimation()
+    }
+    
+    private func setupEmojis(_ imageNames: [String]) {
+        let distance = size.width / CGFloat(imageNames.count)
+
+        for (index, name) in imageNames.enumerated() {
+            let node = SKSpriteNode(imageNamed: name)
+            
+            node.size = CGSize(width: distance*0.7, height: distance*0.7)
+            node.position.y = size.height / 2
+            node.position.x = distance * (CGFloat(index) + 0.5)
+            
+            addChild(node)
+        }
+    }
+    
+    private func startAnimation() {
+        for (index, node) in children.enumerated() {
+            node.run(
```

Comment:
Would it be possible to either split this up to specific animations or ate least put comments? 
It might be quite confusing for a student and if split into methods coding along might be more understandable.

Reply:
I've added a few comments explaining the animation ✅